frontend/pages/tournament/index.njk

```diff
@@ -0,0 +1,15 @@
+{% extends "../templates/base.njk" %}
+
+{% block content %}
+<div id="tournamentData"></div>
+<script>
+    const tournamentId = await router.getTournamentIdParam();
```

Comment:
This code should not have been commited as it was just a demonstration. It is actuall not working. Please remove it.